lib/tasks/cortex/core/media.rake

```diff
@@ -29,15 +29,7 @@ namespace :cortex do
                            },
                          metadata:
                            {
-                             styles: {
-                               large: {geometry: '1800x1800>', format: :jpg},
-                               medium: {geometry: '800x800>', format: :jpg},
-                               default: {geometry: '300x300>', format: :jpg},
-                               mini: {geometry: '100x100>', format: :jpg},
-                               micro: {geometry: '50x50>', format: :jpg},
-                               post_tile: {geometry: '1140x', format: :jpg}
-                             },
-                             processors: [:thumbnail, :paperclip_optimizer],
+                             styles: {},
```

Comment:
Other methods assume styles does exist, which would throw an error otherwise, which is why we should probably have a helper function.I made one for that React cells demo, as well as a in browser text editor as well.

This would allow any config that the user creates to not break.